modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/portlet/shared/task/PortletSharedTaskExecutorImpl.java

```diff
@@ -79,9 +77,8 @@ public <T> T executeOnlyOnce(
 
 	@Activate
 	protected void activate() {
-		String[] requestSharedAttributes = ArrayUtil.append(
-			props.getArray(PropsKeys.REQUEST_SHARED_ATTRIBUTES),
-			"LIFERAY_SHARED_");
+		String[] requestSharedAttributes =
```

Comment:
@lfbesada Here, you can use WebKeys.LIFERAY_SHARED directly. Before, there is no pre-defined prefix, so you get all and sort and use it, now you have one, then use it directly